README.md

```diff
@@ -27,12 +27,12 @@ When building a `Box`, all you need to care about are `Sections`s and `Node`s.
 
 ```swift
 let box = Box<SectionId, RowId>.empty
-                |-+ Section(id: SectionId.user,
-                            header: EmptySpaceComponent(height: 24, color: .clear))
-                |---+ RowId.user <> IconTitleDetailsComponent(icon: image, title: patient.name)
-                |-+ Section(id: SectionId.consultantDate,
-                            header: EmptySpaceComponent(height: 24, color: .clear))
-                |---+ RowId.loading <> LoadingIndicatorComponent(isLoading: true)
+|-+ Section(id: SectionId.user,
+header: EmptySpaceComponent(height: 24, color: .clear))
+|---+ RowId.user <> IconTitleDetailsComponent(icon: image, title: patient.name)
+|-+ Section(id: SectionId.consultantDate,
+header: EmptySpaceComponent(height: 24, color: .clear))
+|---+ RowId.loading <> LoadingIndicatorComponent(isLoading: true)
```

Comment:
It does not look nice 😜
<img width="924" alt="screen shot 2018-05-08 at 11 43 36" src="https://user-images.githubusercontent.com/4622322/39752985-23716a28-52b5-11e8-80cb-4553b8368c26.png">